internal/config/mount_config.go

```diff
@@ -24,9 +24,29 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
+	Size                    int64  `yaml:"size-mb"`
```

Comment:
we should keep keep the yaml names and properties names similar as much as possible.
SizeMB
TTLInSec etc